node/src/BaseClient.ts

```diff
@@ -2218,8 +2221,11 @@ export class BaseClient {
      * Notes:
      * 1. `BRPOP` is a blocking command,
      * see [Blocking Commands](https://github.com/aws/glide-for-redis/wiki/General-Concepts#blocking-commands) for more details and best practices.
-     * 2. When in cluster mode, all `keys` must map to the same `hash slot`.
+     * 2. When in cluster mode, all `keys` must map to the same hash slot.
      *
+     * @remarks
```

Comment:
Looks like we repeat the same thing twice here, once under "Notes" and once under "@remarks", lets delete the "Notes" section